packages/lesswrong/components/posts/RecombeeLatestPosts.tsx

```diff
@@ -218,7 +221,9 @@ const RecombeeLatestPosts = ({ currentUser, classes }: {
     })
   };
 
-  const settingsButton = (<>
+  const showSettingsButton = userIsAdmin(currentUser) || usingClassicLWAlgorithm(selectedScenario);
```

Comment:
What is this intended to be doing?

Reply:
The "customize" button should show on all tabs if you're an admin or just the Latest Posts one if you're not an admin